R/explore_text_columns.R

```diff
@@ -1,19 +1,311 @@
-#' Performs EDA of text features
+#' Performs EDA of text features. Works well with Rmarkdown.
+#' Its highly recommended to use code-chunk option results='asis'
+#' to make use of the markdown syntax
+#' - identifies text columns if not already specified
 #' - prints the summary statistics of character length
 #' - plots the distribution of character length
 #' - prints the summary statistics of word count
 #' - plots the distribution of word count
 #' - plots the word cloud
+#' - plots a bar plot of top 10 words
+#' - plot a word cloud of bigrams
+#' - plot a bar plot of top 10 bigrams
+#'
+#' @importFrom dplyr %>%
 #'
 #' @param df dataframe: dataset
-#' @param text_cols vector of numeric column names
+#' @param text_cols vector of text column names
 #'
-#' @return a list of plots
+#' @return a list of results and plots
+#' @export
 #'
 #' @examples
-#' \dontrun{
-#' explore_text_columns(df)
-#' }
-explore_text_columns <- function(df, text_cols=NULL){
+#' explore_text_columns(cars)
```

Comment:
should we give another example? as there is no text column in cars

Reply:
I've added another example using the "apocrypha" dataset from a developer package [JohnCoene/sacred](https://github.com/JohnCoene/sacred)
I've addressed this is in this [commit](https://github.com/UBC-MDS/datascience.eda.R/pull/15/commits/5d271e4a9a0db4637edfa95e1f331eb4d6a8285a)